### PR TITLE
Now displays room creator in a div at the top of the room page before…

### PR DESCRIPTION
**Pull request by JamieZafar**
… the game starts

### CODE REVIEW

**Comment by thullSL** [comment]
:shipit:

File: src/server/parameters.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
-var answerTime = 5; //answer time length in seconds
-var submitTime = 5;
+var answerTime = 30; //answer time length in seconds
```
Comment (kenneth-gray) [inline src/server/parameters.js]: Handy dev tip: prefer longer names that explain variables over adding comments - keeping comments up to date can cause developers in the future problems.

File: src/client/rooms/room-page.js [hunk 2]
```diff
@@ -59,6 +59,13 @@ export class RoomPage extends Component {
                 <Progress/>
                 <div>
                     <h1>{`Room: ${this.props.roomData.name}`}</h1>
+                    <div className={`col-lg-12 ${style['room-creator']}`}>
```
Comment (kenneth-gray) [inline src/client/rooms/room-page.js]: You can use `style.roomCreator` now :+1:

File: src/client/rooms/room-page.js [hunk 3]
```diff
@@ -83,7 +90,7 @@ function leaveRoom() {
 
 const mapStateToProps = (state, ownProps) => ({
     roomUsers: state.room.users,
-    roomData: state.room.data,
+    roomData: state.room.room,
```
Comment (kenneth-gray) [inline src/client/rooms/room-page.js]: Well this looks weird...

Comment (thullSL) [inline src/client/rooms/room-page.js]: `carData: state.vroom.vroom.vroom`

Comment (JamieZafar) [inline src/client/rooms/room-page.js]: We'll sort this out when we do our front-end refactoring next week if that's okay?

**Comment by JamieZafar** [comment]
:ship: ?

**Pull request merged**